peglib.h

```diff
@@ -1800,27 +1962,27 @@ inline size_t parse_literal(const char* s, size_t n, SemanticValues& sv, Context
         }
     }
 
-	// Word check
-    static Context dummy_c(nullptr, lit.data(), lit.size(), 0, nullptr, nullptr, false, nullptr);
+    // Word check
```

Comment:
Usually I work with tabs, I converted all of them into spaces using Notepad++ and so your rest tabs were converted as well and pop up in the diff now... Sorry :P